nextflow_schema.json

```diff
@@ -261,8 +261,7 @@
                 },
                 "save_merged": {
                     "type": "boolean",
-                    "description": "Save merged reads.",
-                    "default": false
```

Comment:
Why is the default false removed here?

Reply:
I'll update the PR.  Not sure why this got removed.  Thanks for catching this.

Reply:
Updated the PR in https://github.com/nf-core/smrnaseq/pull/494/commits/d7688338b0d38b5ec60e811b8862d49a6d191f18.  Thanks again!